hw/ip/aes/data/aes.hjson

```diff
@@ -337,6 +366,16 @@
           This mode is useful if software needs full control over the AES unit.
         '''
       }
+      { bits: "11",
+        name: "FORCE_ZERO_MASKS",
+        resval: "0"
+        desc:  '''
+          Use masks generated by internal masking PRNG (0) or force all masks constantly to zero (1).
+          Setting all masks to constant zero can be useful when performing SCA analysis.
```

Comment:
nit (pet peeve?): "when performing SCA analysis" -> "when comparing SCA results"
(the A in SCA already means analysis, this is my pet peeve though)

Reply:
Thank you for bearing with me :)